enterprise_access/apps/content_assignments/constants.py

```diff
@@ -11,19 +11,24 @@ class LearnerContentAssignmentStateChoices:
     ACCEPTED = 'accepted'
     CANCELLED = 'cancelled'
     ERRORED = 'errored'
+    REMINDED = 'reminded'
 
     CHOICES = (
         (ALLOCATED, 'Allocated'),
         (ACCEPTED, 'Accepted'),
         (CANCELLED, 'Cancelled'),
         (ERRORED, 'Errored'),
+        (REMINDED, 'Reminded')
     )
 
     # States which allow reallocation by an admin.
     REALLOCATE_STATES = (CANCELLED, ERRORED)
 
     # States which allow cancellation by an admin.
-    CANCELABLE_STATES = (ALLOCATED, ERRORED)
+    CANCELABLE_STATES = (ALLOCATED, ERRORED, REMINDED)
+
+    # States which allow reminders by an admin.
+    REMINDABLE_STATES = (ALLOCATED)
```

Comment:
you need a trailing comma here, otherwise `REMINDABLE_STATES` will just get set to the value of `ALLOCATED`.
for example:
```python
>>> x = 'hello'
>>> foo = (x)
>>> foo
'hello'
>>> for thing in foo: print(thing)
...
h
e
l
l
o
```

Reply:
vs. 
```python
>>> foo = (x,)
>>> for thing in foo: print(thing)
...
hello
>>>
```